tests/test_endpoint_buffers.py

```diff
@@ -0,0 +1,94 @@
+from unittest.mock import MagicMock
+from uuid import uuid4
+
+import pytest
+from gsy_framework.constants_limits import ConstSettings, GlobalConfig
+from pendulum import DateTime, duration
+
+from gsy_e.gsy_e_core.enums import FORWARD_MARKET_TYPES
+from gsy_e.gsy_e_core.sim_results.endpoint_buffer import SimulationEndpointBuffer
+
+
+@pytest.fixture(name="forward_setup")
+def forward_setup_fixture():
+    """Create area with all the forward markets and
+    pre added timeslots for each forward market."""
```

Comment:
Not really. I break some lines to make it visually more readable when it's too long. But this one seems not to be so long, so I reverted the line break for you.

Reply:
(We also had a conversation about whether to use 80 vs 100 characters. In the end the team preferred using 100 since new displays make it possible without too many complications).